src/data/stream/init.lean

```diff
@@ -25,7 +25,7 @@ instance {α} [inhabited α] : inhabited (stream α) :=
 protected theorem eta (s : stream α) : head s :: tail s = s :=
 funext (λ i, begin cases i; refl end)
 
-theorem nth_zero_cons (a : α) (s : stream α) : nth (a :: s) 0 = a := rfl
+@[simp] theorem nth_zero_cons (a : α) (s : stream α) : nth (a :: s) 0 = a := rfl
```

Comment:
It's unfortunate that these lemmas are named backwards (with respect to e.g. `list.nthd_cons_zero`). Might be worth fixing in another PR.